### ISSUE TITLE
Ability to hide join and part messages

### ISSUE THREAD

**Issue by orsenthil** [mephux/komanda#243]
I did not see a feature request already open, but an ability to hide join and part messages and plus a hide messages based on a regex match will be helpful for an IRC client. Komanda will definitely benefit as the interfaces look so clean. Thank you!

**Comment by ghost** [comment]
Komanda can already hide join/part's since f9543cf06bba89608c5628b9043c9ed2641ebb8a. Assuming you built komanda from source or have some nightly/devel build,  Gear's icon on the bottom left -> Display... but the regex thing does not exist.

**Comment by orsenthil** [comment]
Are there any beta images available? I tried to run it from master, but constantly ran into failures.

```
Running tasks: run:mac

Running "run:mac" task

Running "default" task

Running "shell:runnw" (shell) task
Verifying property shell.runnw exists in config...OK
File: [no files]
Options: stdout=false, stderr=false, stdin=false, failOnError
Command: ./build/cache/0.10.2/osx/node-webkit.app/Contents/MacOS/node-webkit . > /dev/null 2>&1
Warning: Command failed:  Use --force to continue.

```

**Comment by qqndrew** [comment]
Yes ... this particular issue is currently being fixed in https://github.com/mephux/komanda/pull/241, although I've run into some trouble with the tarring task, the compiled stuff should still show up inside the /build/Komanda folder as of the current latest commit

fair warning: I do plan on force pushing/merging prior to marking the pr as mergable in case you are planning on doing any dev work on that branch yourself
